proto/cosmos/auth/query.proto

```diff
@@ -8,29 +8,29 @@ import "cosmos_proto/cosmos.proto";
 
 option go_package = "github.com/cosmos/cosmos-sdk/x/auth/types";
 
-// Query creates service with Account and Parameters as rpc
+// Query defines the gRPC querier service.
 service Query{
-    // Account returns account details based on address
+    // Account returns account details based on address.
     rpc Account (QueryAccountRequest) returns (QueryAccountResponse) {}
 
-    // Params queries all parameters
+    // Params queries all parameters.
     rpc Params (QueryParamsRequest) returns (QueryParamsResponse) {}
 }
 
-// QueryAccountRequest is request type for the Query/Account RPC method
+// QueryAccountRequest is the request type for the Query/Account RPC method.
 message QueryAccountRequest{
-    bytes address = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+  required bytes address = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
```

Comment:
Are we okay to add `required` fields in these objects? They really are required.

Only thing holding me back is:
https://developers.google.com/protocol-buffers/docs/proto#specifying_field_rules

> Required Is Forever You should be very careful about marking fields as required. If at some point you wish to stop writing or sending a required field, it will be problematic to change the field to an optional field – old readers will consider messages without this field to be incomplete and may reject or drop them unintentionally. You should consider writing application-specific custom validation routines for your buffers instead. Some engineers at Google have come to the conclusion that using required does more harm than good; they prefer to use only optional and repeated. However, this view is not universal.